target/linux/ipq806x/image/generic.mk

```
@@ -101,6 +113,21 @@ define Device/arris_tr4400-v2
 endef
 TARGET_DEVICES += arris_tr4400-v2
 
+define Device/aruba_ap-325
+	$(call Device/LegacyImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := Aruba
+	DEVICE_MODEL := AP-325
+	SOC := qcom-ipq8068
+	PAGESIZE := 2048
+	BLOCKSIZE := 128k
```

Does it make sense to define a limit for max image size?
```suggestion
	BLOCKSIZE := 128k
	IMAGE_SIZE := 65536k
```

---

also does anyone know when to add `UBINIZE_OPTS := -E 5` and when not?